exposure-notification/README.md

```diff
@@ -275,8 +276,8 @@ Status API is a one-call replacement for diagnosis key current & list fetches as
           "transmissionRiskLevel": 5,
           "rollingStartIntervalNumber":  123456,
           "rollingPeriod": 144,
-          "visitedCountries": "DE",
```

Comment:
Why are these here for every key? Shouldn't they be once for the whole body?

Reply:
Yes, it should.